bambou/session.go

```diff
@@ -121,14 +130,25 @@ func (s *Session) makeAuthorizationHeaders() (string, *Error) {
 
 func (s *Session) prepareHeaders(request *http.Request, info *FetchingInfo) *Error {
 
-	authString, berr := s.makeAuthorizationHeaders()
-	if berr != nil {
-		return berr
+	if s.Certificate != nil { // We're using X509 certificate based auth.
+
+		// XXX - "InsecureSkipVerify"
```

Comment:
Can we replace this with `// Skipping TLS certificate verification`, XXX comments do not look good